FontAwesome5.js

```diff
@@ -0,0 +1,133 @@
+/**
```

Comment:
Think you should separate this out to its own file just for the factory function.